app/_src/production/cp-deployment/systemd.md

```diff
@@ -3,21 +3,21 @@ title: Systemd
 content_type: how-to
 ---
 
-When using {{site.mesh_product_name}} on VMs it is recommended to use a process manager like systemd.
+When using {{ site.mesh_product_name }} on VMs it is recommended to use a process manager like systemd.
 Here are examples of systemd configurations
 
 ## kuma-cp
 
-```
+```systemd
 [Unit]
-Description = Kuma Control Plane
+Description = {{ site.mesh_product_name }} Control Plane
 After = network.target
-Documentation=https://kuma.io
+Documentation={{ site.links.web }}
 
 [Service]
-User=kuma
-Environment=KUMA_MODE=standalone
-ExecStart = /path/to/kuma/bin/kuma-cp run --config-file=/home/kuma/cp-config.yaml
+User = {{ site.mesh_product_name_lowercase }}
```

Comment:
as discussed on slack should we just use `{{ site.mesh_product_name | downcase }}`?

Reply:
I actually like `| downcase` as it doesn't involve in making sure this variable is set in other places.

Reply:
Addressed